about/about-networking.md

```diff
@@ -0,0 +1,152 @@
+---
+title: About Networking
+description: Learn about networking!
+---
+
+You can get up and running with Calico by following any of our getting started guides without needing to be a networking
```

Comment:
Please link "getting started guides".  With the change to the top level nav structure, this page is one of the most prominent in the whole site.  A new visitor could easily end up on this page straight after "About Calico".